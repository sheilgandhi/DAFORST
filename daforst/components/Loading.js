import { StyleSheet, ActivityIndicator, View } from 'react-native';
import React from 'react';
import { COLORS } from '../colors';

const Loading = () => {
  return (
    <View style={styles.container}>
      <ActivityIndicator size="large" color={COLORS.green} />
    </View>
  );
};

export default Loading;

const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
    marginTop: 30,
  },
});
