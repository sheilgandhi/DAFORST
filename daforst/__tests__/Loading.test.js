import React from 'react';
import renderer from 'react-test-renderer';
import Loading from '../components/Loading';

describe('Loading', () => {
  test('renders correctly', () => {
    const tree = renderer.create(<Loading />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
