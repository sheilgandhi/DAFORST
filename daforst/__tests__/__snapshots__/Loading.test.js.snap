// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
      "marginTop": 30,
    }
  }
>
  <ActivityIndicator
    color="#55897B"
    size="large"
  />
</View>
`;
